ast.go

```diff
@@ -43,12 +43,14 @@ func (g *graph) Lookup(name string) (Entity, bool) {
 
 // ProcessDescriptors is deprecated; use ProcessCodeGeneratorRequest instead
 func ProcessDescriptors(debug Debugger, req *plugin_go.CodeGeneratorRequest) AST {
-	return ProcessCodeGeneratorRequest(debug, req)
+	return ProcessCodeGeneratorRequest(debug, req, false)
 }
 
 // ProcessCodeGeneratorRequest converts a CodeGeneratorRequest from protoc into a fully
-// connected AST entity graph. An error is returned if the input is malformed.
-func ProcessCodeGeneratorRequest(debug Debugger, req *plugin_go.CodeGeneratorRequest) AST {
+// connected AST entity graph. Set bidirectional to True for the AST to be built with
+// a complete list of a given entity's dependents rather than a shallow list. An error
+// is returned if the input is malformed.
+func ProcessCodeGeneratorRequest(debug Debugger, req *plugin_go.CodeGeneratorRequest, bidirectional bool) AST {
```

Comment:
Is there a way we can do this without breaking the API?

Reply:
that's true, i could make a method that's called on the AST that adds bidirectionality afterwards